### DISCUSSION TITLE
[General] Error when performing step 8.1 'Manually relaying interop messages with cast' in Supersim

### DISCUSSION THREAD

**Discussion by wispyiwnl** [ethereum-optimism/developers#610] [General]
### Discussion Topic

I’m testing Supersim and currently on Step 8.1, 'Manually relaying interop messages with cast' from the guide, but when I reach step 7, I encounter the following error:

```
Error:
server returned an error response: error code 3: execution reverted: custom error 0xeda86850, data: "0xeda86850"
```

This is the command I’m running:

```
cast send 0x4200000000000000000000000000000000000022 \
"executeMessage((address, uint256, uint256, uint256, uint256), address, bytes)" \
"(0x4200000000000000000000000000000000000023, 11, 1, 1728057826, 901)" \
0x4200000000000000000000000000000000000023 \
0x000000000000000000000000420beef00000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000064d9f50046000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb92266000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb9226600000000000000000000000000000000000000000000000000000000000003e800000000000000000000000000000000000000000000000000000000 \
--rpc-url http://127.0.0.1:9546 \
--private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```
I've already customized the command with the `blockNumber`, `logIndex`, `timestamp` and `data`.

I'm not sure if the error is due to something I'm doing wrong or if there's an issue with the tool

Here is the link where I'm following the step-by-step guide: https://supersim.pages.dev/guides/interop/manually-relaying-interop-messages-cast#7-send-the-relaymessage-transaction

**Comment by opfocus** [comment]
Please be patient, I'd like to try running it as well, I'll be back here with any progress

**Comment by tremarkley** [comment]
@wispyiwnl just updated example 8.1 to reflect the latest changes to interop. Can you follow the updated steps: https://supersim.pages.dev/guides/interop/manually-relaying-interop-messages-cast#manually-relaying-interop-messages-with-cast

**Reply by wispyiwnl** [reply]
Yes, it’s working now. Thank you very much!
